packages/core/stories/overlay/overlay.stories.tsx

```diff
@@ -137,14 +138,6 @@ export const LongContent = () => {
   );
 };
 
-const Divider = () => {
-  return (
-    <div className="divider-container">
```

Comment:
Does this mean there are extra styles to remove

Reply:
removed